libbeat/docs/monitoring/monitoring-beats.asciidoc

```diff
@@ -1,74 +1,28 @@
-//// include::../../libbeat/docs/monitoring/configuring.asciidoc[]
-//// Make sure this content appears below a level 2 heading.
-//////////////////////////////////////////////////////////////////////////
-
 [role="xpack"]
 [[monitoring]]
 = Monitoring {beatname_uc}
 
 [partintro]
 --
 
-NOTE: The {monitor-features} for {beatname_uc} require {es} {beat_monitoring_version} or later.
-
-The {stack} {monitor-features} enable you to easily monitor {beatname_uc} from {kib}. For more
-information, see
-{stack-ov}/xpack-monitoring.html[Monitoring the {stack}] and
-{kibana-ref}/beats-page.html[Beats Monitoring Metrics].
+You can use the {stack} {monitor-features} to gain insight into the health of
+{beatname_uc} agents running in your environment. 
 
-To configure {beatname_uc} to collect and send monitoring metrics:
+To monitor {beatname_uc}, make sure monitoring is enabled on your {es} cluster,
+then configure the method used to collect {beatname_uc} metrics. You can use one
+of following methods:
 
-. Create a user that has appropriate authority to send system-level monitoring
-data to {es}. For example, you can use the built-in +{beat_monitoring_user}+ user or
-assign the built-in +{beat_monitoring_user}+ role to another user. For more
-information, see
-{stack-ov}/setting-up-authentication.html[Setting Up User Authentication] and
-{stack-ov}/built-in-roles.html[Built-in Roles].
+* <<monitoring-internal-collection,Internal collection>> - Internal
+collectors send monitoring data directly to your monitoring cluster.
+* <<monitoring-internal-collection-legacy,Legacy internal collection (deprecated)>> - 
+Legacy internal collectors send monitoring data to your production cluster.
 
-. Add the `monitoring` settings in the {beatname_uc} configuration file. If you
-configured the {es} output and want to send {beatname_uc} monitoring events to
-the same {es} cluster, specify the following minimal configuration:
-+
-["source","yml",subs="attributes"]
---------------------
-monitoring:
-  enabled: true
-  elasticsearch:
-    username: {beat_monitoring_user}
-    password: somepassword
---------------------
-+
-If you configured a different output, such as {ls} or you want to send {beatname_uc}
-monitoring events to a separate {es} cluster (referred to as the _monitoring cluster_),
-you must specify additional configuration options. For example:
-+
-["source","yml",subs="attributes"]
---------------------
-monitoring:
-  enabled: true
-  elasticsearch:
-    hosts: ["https://example.com:9200", "https://example2.com:9200"] <1>
-    username: {beat_monitoring_user}
-    password: somepassword
---------------------
-<1> This setting identifies the hosts and port numbers of {es} nodes
-that are part of the monitoring cluster.
 
-. {kibana-ref}/monitoring-xpack-kibana.html[Configure monitoring in {kib}].
+To learn about monitoring in general, see 
+{stack-ov}/xpack-monitoring.html[Monitoring the {stack}]. 
```

Comment:
I don't see this fragment in `master`. Do we want it in this PR?

Reply:
We commented out this link in master because the docs in the ES Reference are likely to add confusion (rather than help) until we get them updated. I think it's unlike that we'll make improvements to the 7.2 version of the docs, so let's leave the link in here.